### PR TITLE
Update cam trigger page

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-pixhawk-camera-trigger-setup.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ Items Required
 ==============
 
 -  Sony A6000 (or other camera with wire shutter release)
--  ArduPilot compatible autopilot
+-  ArduPilot compatible autopilot with a free GPIO pin. See :ref:`common-gpios` for more information
```
Comment (rmackay9) [inline common/source/docs/common-pixhawk-camera-trigger-setup.rst]: I think if we are talking about the autopilot controlling a GPIO pin we should actually stick with the word "Relay".

File: common/source/docs/common-pixhawk-camera-trigger-setup.rst [hunk 2]
```diff
@@ -84,25 +84,20 @@ e) Insert the Servo Pins into a Servo connector, one pin on each side of the mid
...
 
 a) Connect to your Autopilot via Mission Planner either through direct USB connection or wireless telemetry link.
 
-b) Connect your modified shutter release cable to AUX 1 on the Pixhawk
-       (CH9 on APM) making sure to maintain the correct orientation for
-       polarity.
-
-.. image:: ../../../images/CTimage11.png
-    :width: 450px
+b) Connect your modified shutter release cable to a free GPIO capable pin on the autopilot making sure to maintain the correct orientation for polarity. For example, by default on the Pixhawk, AUX5 or AUX6 is configured to be a GPIO.
```
Comment (rmackay9) [inline common/source/docs/common-pixhawk-camera-trigger-setup.rst]: Let's say relay

**Comment by rmackay9** [comment]
I've made a couple of comments but it's overall an improvement so merged, thanks!

**Comment by Hwurzburg** [comment]
added some wording re use as relay pins

**Pull request merged**